internal/seetie/convert.go

```diff
@@ -188,6 +185,7 @@ func ValueToStringSlice(src cty.Value) []string {
 		return l
 	}
 
+	l = []string{}
```

Comment:
Same here, if there is no append, no one will iterate over this list. Obsolete alloc.